app/controllers/answers/getAnswerById.js

```diff
@@ -0,0 +1,41 @@
+import { DEFAULT_ERROR_MESSAGE } from '~/utils/backend/constants';
+import { INVALID_PARAMS_FOR_OPERATION_ERROR_MESSAGE } from '~/utils/constants';
+import { db } from '~/utils/db.server';
+
+export const getAnswerById = async (answerId, user) => {
+  if (!answerId || typeof answerId !== 'number' || parseInt(answerId) < 1) {
+    return {
+      error: {
+        message: INVALID_PARAMS_FOR_OPERATION_ERROR_MESSAGE,
+        detail: 'The answer id must be an integer not minor to 1',
+      },
+    };
+  }
+
+  if (!user) {
+    return {
+      error: {
+        message: INVALID_PARAMS_FOR_OPERATION_ERROR_MESSAGE,
+        detail: 'Please provide the user object',
+      },
+    };
+  }
+
+  try {
+    const answer = await db.Answers.findUniqueOrThrow({
+      where: { answer_id: answerId },
+    });
+
+    return {
+      success: true,
```

Comment:
This return.

Reply:
Good spot, didnt see the boolean success response. 

@Hoosep For context, we return as strings right now to be able to use this component to show toastify success/error messages dynamically https://github.com/wizeline/wize-q-remix/blob/master/app/components/Notifications/Notifications.jsx

Reply:
Pretty nice! I love it. Thank you so much for context.

Reply:
@AlonsoMtzG It just changes the boolean success for a string.